packages/components/src/Modal/GenericModal/_docs/GenericModal.stories.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react"
+import { Meta, StoryObj } from "@storybook/react"
+import { GenericModal } from "../index"
+
+const meta = {
+  title: "Components/Modals/Generic Modal",
+  component: GenericModal,
+  args: {
+    children: "Example content",
+    isOpen: false,
+  },
+} satisfies Meta<typeof GenericModal>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const Playground: Story = {
+  render: ({ isOpen: propsIsOpen, ...args }) => {
+    const [isOpen, setIsOpen] = React.useState<boolean>(propsIsOpen)
+    const handleDismiss = (): void => setIsOpen(false)
+
```

Comment:
Suggestion:
If you want to be able to have `isOpen` change according to the SB controls, you can add a `useEffect` to watch for a change to `args.isOpen`. Might not be applicable given focus lock, but just FYI :)

Reply:
Ah that does make sense, thanks